Makefile

```diff
@@ -8,5 +8,6 @@ deps:
 	luarocks install luasocket
 	luarocks install lua-cjson
 	luarocks install uuid
+	luarocks install luasec
```

Comment:
should this go into the rockspec too? it's optional, so I'm not sure

Reply:
I would prefer to keep the `luasec` dependency optional, which I guess means `no`. Adding it to the `Makefile` is fine since it is only used for development, where you clearly want to be able to test that you are not breaking the TLS support.

Reply:
makes sense!